Directory.Build.props

```diff
@@ -15,7 +15,7 @@
   <!-- GLOBALLY USABLE VARIABLES -->
   <PropertyGroup>
     <!-- Establish a preset but make it an active choice for each project. -->
-    <SupportedNonMetaTargetFrameWorks>netcoreapp2.2;netcoreapp3.0</SupportedNonMetaTargetFrameWorks>
+    <SupportedNonMetaTargetFrameWorks>netcoreapp3.0</SupportedNonMetaTargetFrameWorks>
```

Comment:
The reason why I chose to remove the use of `$(SupportedNonMetaTargetFrameWorks)` in `Demo.csproj` is that it doesn't make sense for a website to target two runtimes at the same time. However, I think it makes sense that `$(SupportedNonMetaTargetFrameWorks)` does contain all the runtimes the library is supposed to support. This list might be useful in several places, while the runtime used by the demo website isn't.
A nice side effect is that the unit tests will run twice (once for every runtime), which will make sure that the library stays compatible with `netcoreapp2.2` and `netcoreapp3.0`.
I think we should revert the changes in `Demo.csproj` and `Directory.Build.props` in order to keep unit testing on both runtimes.

Reply:
I couldn't get the project to work on both runtimes on all platforms at the same time time.

Reply:
Was it not working on your environment or on the CI build? I had no problem on my end.

Reply:
It was in my environment. I will run through it again.